R/chat.R

```diff
@@ -313,9 +313,18 @@ Chat <- R6::R6Class(
     #'   that yields strings. While iterating, the generator will block while
     #'   waiting for more content from the chatbot.
     #' @param ... The input to send to the chatbot. Can be strings or images.
-    stream = function(...) {
+    #' @param content Whether the stream should yield only `"text"` or `"all"`
+    #'   content types created during the round. When `content = "all"`,
+    #'   `stream()` yields [Content] objects.
+    stream = function(..., content = c("text", "all")) {
```

Comment:
How about `yield = c("text", "content")`?

Reply:
I'm okay with `yield` as the argument name (in ellmer) but one reason I didn't use it is because it's a reserved keyword in Python, meaning that chatlas can't use it even as an argument name. (And Carson has already implemented the `content="all"` argument in chatlas.)

Reply:
Another reason I didn't love `yield` is that you don't necessarily need to know what a generator is to use `$stream()` (especially in the Shiny case).

Reply:
Hmmm, I'm open to names other than `yield` but I do think the key distinction is between whether you yield text or Content objects.

Reply:
How about `results = c("text", "all")`? Or `stream`?

Reply:
I like `results = c("text", "content")`. I like keeping the word "content" in the options, connecting the `"content"` to the `Content` objects that you'll get as a result.

Reply:
Okay, final answer: `stream = c("text", "content")`. It's a little awkward (`chat$stream(stream = "content")`) but it's the closest to `yield` and this isn't expected to be a commonly used option beyond shinychat.